e2e/helpers/api.helper.ts

```diff
@@ -17,8 +17,8 @@ export class ApiHelper {
         return this.makeRequest<TResponseData>({
             method: 'get',
             url: `${this.baseUrl}${path}`,
-            params,
             validateStatus: () => true,
+            ...params,
```

Comment:
`params` wasn't supposed to be used like this, but if we are doing this then please also update the type to `AxiosRequestConfig`